vulnerabilities/scraper/npm.py

```diff
@@ -47,30 +47,27 @@ def remove_spaces(x):
     return x
 
 
-def get_all_version(package_name):
+def get_all_versions(package_name):
     """
-    Returns all available for a module
+    Returns all versions available for a module
     """
-    package_url = NPM_URL.format('/'+package_name)
-    response = urlopen(package_url).read()
-    data = json.loads(response)
-    versions = data.get('versions', {})
-    all_version = [obj for obj in versions]
-    return all_version
+    package_url = NPM_URL.format(f'/{package_name}')
+    data = json.load(urlopen(package_url))
+    return [v for v in data.get('versions', {})]
 
 
-def extract_version(package_name, aff_version_range, fixed_version_range):
+def extract_versions(package_name, aff_version_range, fixed_version_range):
     """
-    Seperate list of Affected version and fixed version from all version
-    using the range specified
+    Seperate list of affected versions and fixed versions from all version
+    using the ranges specified
     """
-
+    # FIXME This skips unfixed vulnerabilities
```

Comment:
Good catch... this warrants a ticket

Reply:
Right. I meant to create one for it but apparently forgot. :-/ Done that now: https://github.com/nexB/vulnerablecode/issues/126